src/main.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file   main.cpp
+ * @author Dominik Authaler
+ * @date   01.04.2020 (creation)
+ */
+
+#include "CLI/CLI.hpp"
+#include "spdlog/spdlog.h"
+constexpr unsigned int maxVerbosity     =  5;
+constexpr unsigned int defaultVerbosity =  1;
+constexpr unsigned int defaultPort      = 17;
+
+int main(int argc, char *argv[]) {
+    CLI::App app;
+
+    std::string  characterPath;
+    std::string  matchPath;
+    std::string  scenarioPath;
+
+    unsigned int verbosity = defaultVerbosity;
+    unsigned int port      = defaultPort;
+
+    std::vector<std::string> additionalOptions;
```

Comment:
ist so verlangt, würde das aber später gerne in einzelne optionen auslagern, das macht benutzung und validieren leichter

Reply:
Mir gefällt das auch nicht, aber der Standard schreibt es halt genau so vor.